helm/mds/templates/deployment.yaml

```diff
@@ -1,37 +1,38 @@
-{{- range $api := .Values.apis }}
+{{- range $key, $api := .Values.apis }}
+{{- if $api.enabled }}
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ $api.name }}
+  name: {{ $key }}
   labels:
-    app: {{ $api.name }}
+    app: {{ $key }}
   namespace: {{ $.Release.Namespace }}
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: {{ $api.name }}
+      app: {{ $key }}
   template:
     metadata:
       labels:
-        app: {{ $api.name }}
+        app: {{ $key }}
     spec:
       containers:
-      - name: {{ $api.name }}
+      - name: {{ $key }}
         {{- if $.Values.registry }}
-        image: {{ $.Values.registry }}/{{ $api.name }}:{{ $api.version }}
+        image: {{ $.Values.registry }}/{{ $key }}:{{ $api.version }}
         {{- else }}
-        image: {{ $api.name }}:{{ $api.version }}
+        image: {{ $key }}:{{ $api.version }}
         imagePullPolicy: IfNotPresent
         {{- end }}
         resources:
           limits:
-            cpu: 500m
-            memory: 512Mi
+            cpu: {{ $.Values.resourcesLimitsCpu | default "500m" }}
+            memory: {{ $.Values.resourcesLimitsMemory | default "512Mi" }}
           requests:
-            cpu: 50m
-            memory: 128Mi
+            cpu: {{ $.Values.resourcesRequestsCpu | default "50m" }}
+            memory: {{ $.Values.resourcesRequestsMemory | default "128Mi" }}
```

Comment:
As long as we're doing this - I think it would be best to have the resources be set per-API rather than common across all APIs.  It's reasonable to expect that say mds-provider might have different requirements than mds-policy-author.

Reply:
So `{{ $api.resourcesLimitsCpu }}`, `{{ $api.resourceLimitsMemory }}`, etc.

Reply:
agreed. i have been looking at providing api-service-level resource overrides and ran into blockers given we are iterating over the api-services via a range function. turns out stepping away form the keyboard for a bit is "a good thing(tm)" and with a fresh approach i was able to push a change that provides, in order of precedence, specifying resource values at the api-service level, at the fleet (aka all services) and, lastly, the default (in template) value. eg:

```
# note: override mds-agency resources
% ... --set api.[api-service].[resourcesLimits[Cpu|Memory]=[X]|resourcesRequests[Cpu|Memory]=[Y]]
# note: override all service resources
% ... --set resourcesLimits[Cpu|Memory]=[X]|resourcesRequests[Cpu|Memory]=[Y]
```

note: these configurations can be used together, eg override all defaults while specifying distinct values per service